demisto_sdk/commands/common/docker_images_metadata.py

```diff
@@ -0,0 +1,101 @@
+import re
+from enum import Enum
+from typing import Dict, Optional
+
+from packaging.version import Version
+from pydantic import BaseModel
+
+from demisto_sdk.commands.common.constants import (
+    DOCKERFILES_INFO_REPO,
+)
+from demisto_sdk.commands.common.git_content_config import GitContentConfig
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.common.singleton import PydanticSingleton
+from demisto_sdk.commands.common.tools import get_remote_file_from_api
+
+DOCKER_IMAGES_METADATA_NAME = "docker_images_metadata.json"
+
+# regex to extract docker-images that are specific to content / dockerfiles
+DOCKER_IMAGE_REGEX = r"^demisto/([^\s:]+):(\d+(\.\d+)*)$"
```

Comment:
are there images not starting with 'demisto/'? 
such as here:
https://github.com/demisto/demisto-sdk/pull/3339/files#diff-3ff89907faea6587886223530c2f5d6bba10bda00f22b4a610481f359fc96ef4R27

Reply:
this regex is used to extract only docker-images related to **content** and not custom docker-images.
See the comment in line 18.

I have changed the name of the variable so it would be more clear.